README.md

```diff
@@ -58,8 +35,91 @@ A Gutter Game is when the player never hits a pin (20 zero scores).
 
 A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
 
-In the image below you can find some score examples.
-
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
-
-![Ten Pin Score Example](images/example_ten_pin_scoring.png)
+---
+
+### Modelling
+
+|     class      |   initialize                   |   method                                       |
+|----------------|--------------------------------|------------------------------------------------|
+|   BowlingGame  |  @current_frame = 1            | roll_1(players_score)                          |
+|                |  @strike = false               | roll_2(players_score)                          |
+|                |  @spare = false                | roll_3(players_score)                          |
+|                |  @scorecard = Scorecard.new    | roll_2(players_score)                          |
+|                |                                | next_frame                                     |
+|                |                                | update_roll_1_score(players_score)             |
+|                |                                | update_roll_2_score(players_score)             |
+|                |                                | update_roll_3_score(players_score)             |
+|                |                                | update_bonus(players_score)                    |
+|                |                                | update_scorecard                               |
+|                |                                | view_scorecard                                 |
+|                |                                | end_of_game                                    |
+|  Scorecard     |   @scorecard = []              | update_bonus(players_score)                    |
+|                |   @roll_1_score = 0            | update_scorecard(current_frame, strike, spare) |
+|                |   @roll_2_score = 0            | running_total(current_frame)                   |
+|                |   @roll_3_score = nil          |                                                |
+|                |   @bonus_score = 0             |                                                |
+|                |   @total_game_score = 0        |                                                |
+
+
+### irb
```

Comment:
Really nice layout of how your program works in IRB. I find it's really useful to have this before looking at the ruby file, for better understanding!